bin/satpy_launcher.py

```diff
@@ -78,7 +78,11 @@ def process(msg, prod_list):
     job = message_to_job(msg, config)
     for wrk in config['workers']:
         cwrk = wrk.copy()
-        cwrk.pop('fun')(job, **cwrk)
+        try:
+            func = wrk('fun')
+            func(job, **cwrk)
+        except AbortProcessing:
+            LOG.info("Processing aborted by '%s'", func.__name__)
```

Comment:
don't we want to print the message too, ? like
```suggestion
            LOG.info("Processing aborted by '%s': %s", func.__name__, str(err))
```

Reply:
The plugin should already have logged the reason, so don't think duplicate message is needed.